src/core/support/thd_win32.c

```diff
@@ -61,7 +61,7 @@ struct thd_info {
 static thread_local struct thd_info *g_thd_info;
 
 /* Destroys a thread info */
-static destroy_thread(struct thd_info *t) {
```

Comment:
I guess we should.

On Wed, Apr 8, 2015 at 4:29 PM, Craig Tiller <notifications@github.com>
wrote:

> In src/core/support/thd_win32.c
> <https://github.com/grpc/grpc/pull/1231#discussion_r28024095>:
>
> > @@ -61,7 +61,7 @@ struct thd_info {
> >  static thread_local struct thd_info *g_thd_info;
> >
> >  /* Destroys a thread info */
> > -static destroy_thread(struct thd_info *t) {
>
> We should probably crank up the severity on Visual Studio
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/1231/files#r28024095>.
>